static/js/web-install.js

```diff
@@ -299,4 +306,12 @@ if ("usb" in navigator) {
     console.log("WebUSB unavailable");
 }
 
+// This will create an alert box to stop the user from leaving the page during actions
+window.addEventListener("beforeunload", stopUser => {
```

Comment:
Prefer naming `stopUser` as `event` by convention since it's the event being handled.